### PR TITLE
Schema: Update notebook

### PR DESCRIPTION
**Pull request by haneslinger**

### CODE REVIEW

**Review by JieXiong9119 — COMMENTED** [review]
The codes look good. Do we plan to leave the description part to a new PR or wait for @nllong?

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 1]
```diff
@@ -0,0 +1,1435 @@
...
+   "id": "d149e7ad",
+   "metadata": {},
+   "source": [
+    "#### ResourceUses and TimeSeries Data\n",
+    "Now that we have a current building measured scenario, we want to declare energy and monthly billing data. Per Std 211 6.1.2.1, a minimum of 12 months (preferably up to 3 years) of energy use data is required. The mechanical system of the small office prototype is a heatpump air handler with natural gas backup. We know that Missoula is cold, so likely it will use natural gas backup at some point during its operation. We run an example simulation to get estimates for this, which come out as follows:\n",
+    "\n",
+    "| Resource Type | Jan | Feb | Mar | Apr | May | Jun | Jul | Aug | Sep | Oct | Nov | Dec |\n",
+    "| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |\n",
+    "| Electricity (kWh) | 6792.89 | 5841.75 | 6025.19 | 4985.3 | 5184.04 | 5358.55 | 5755.67 | 5981.78 | 5401.94 | 5225.84 | 5672.15 | 6291.63 |\n",
+    "| Natural Gas (MMBtu) | 5.7 | 4.01 | 0.58 | 0.4 | 0.02 | 0 | 0 | 0 | 0 | 0.01 | 0.36 | 6.08 | 17.16 |\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:670]:
```suggestion
    "| Natural Gas (MMBtu) | 5.7 | 4.01 | 0.58 | 0.4 | 0.02 | 0 | 0 | 0 | 0 | 0.01 | 0.36 | 6.08 | 17.16 |\n",
    "| GHG Emissions (MtCO2e) | 250.0 | 240.0 | 260.0 | 250.0 | 260.0 | 230.0 | 280.0 | 270.0 | 260.0 | 250.0 | 240.0 | 250.0 |\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 2]
```diff
@@ -0,0 +1,1435 @@
...
+    ")\n",
+    "\n",
+    "# given the above, we add the annual totals\n",
+    "elec_ru += bsync.AnnualFuelUseNativeUnits(68516.73)\n",
+    "elec_ru += bsync.AnnualFuelUseConsistentUnits(234.) #convert to MMBTU\n",
+    "elec_ru += bsync.AnnualPeakNativeUnits(21.12) # kW as specified above\n",
+    "elec_ru += bsync.AnnualPeakConsistentUnits(21.12) # already in kW, same as above\n",
+    "elec_ru += bsync.AnnualFuelCost(5304.) # $28 monthly + $0.0725/kwh per https://missoulaelectric.com/member-care/billing-payment/rates/\n",
+    "\n",
+    "# create a resource use for electricity, units of kWh, all end uses\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:705]:
```suggestion
    "# create a resource use for greenhouse gas emissions, units of MtCO2e, all end uses\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 3]
```diff
@@ -0,0 +1,1435 @@
...
+    "# we also add info about the 'peak' units\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:706]:
```suggestion

```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 4]
```diff
@@ -0,0 +1,1435 @@
...
+    "# we also connect it up to a utility\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:707]:
```suggestion

```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 5]
```diff
@@ -0,0 +1,1435 @@
...
+    "ghg_linked_ids = bsync.EmissionsLinkedTimeSeriesIDs()\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:708]: @JieXiong9119  should this section (lines 708-711) be included similar to how it's done for lines 747-748?

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 6]
```diff
@@ -0,0 +1,1435 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3a306b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a new TimeSeries element\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:818]:
```suggestion
    "# Create a new TimeSeries element now for GHG Emissions\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 7]
```diff
@@ -0,0 +1,1435 @@
...
+    "# [ start, end ) \n",
+    "jan_ghg_elec += bsync.StartTimestamp(start_dt)\n",
+    "jan_ghg_elec += bsync.EndTimestamp(end_dt)\n",
+    "\n",
+    "# next, we convey the IntervalReading, i.e. the value.\n",
+    "# The value should use the same units as declared by the ResourceUse/ResourceUnits\n",
+    "jan_ghg_elec += bsync.IntervalReading(250.0)\n",
+    "\n",
+    "# finally, we need to convey to which resource this timeseries element is 'connected',\n",
+    "# i.e. the value represented by this reading conveys information for the Electricity resource use\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:848]:
```suggestion
    "# i.e. the value represented by this reading conveys information for the GHG Emissions resource use\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 8]
```diff
@@ -0,0 +1,1435 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72ac1ee1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# all timeseries is captured in a TimeSeriesData parent\n",
+    "ts_data = bsync.TimeSeriesData()\n",
+    "\n",
+    "# Create a new TimeSeries element\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:773]:
```suggestion
    "# Create a new TimeSeries element for Electricity\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 9]
```diff
@@ -0,0 +1,1435 @@
...
+    "    ),\n",
+    "    ID=\"Scenario-Target\"\n",
+    ")\n",
+    "target_st = bsync.Scenario.ScenarioType()\n",
+    "target = bsync.Target(\n",
+    "    bsync.ReferenceCase(IDref=bench_sc[\"ID\"]),\n",
+    "    bsync.AnnualSavingsSiteEnergy(67181.5),\n",
+    "    bsync.AnnualSavingsCost(931),\n",
+    "    bsync.ENERGYSTARScore(70.),\n",
+    "    bsync.AnnualSavingsMarginalGHGEmissions(200.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:1136]:
```suggestion
 
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 10]
```diff
@@ -0,0 +1,1435 @@
...
+    "    bsync.AnnualSavingsAverageGHGEmissions(500.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:1137]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 11]
```diff
@@ -0,0 +1,1435 @@
...
+    "            ),\n",
+    "            bsync.CostCategory(\"Capital\"),\n",
+    "            bsync.SimpleImpactAnalysis(\n",
+    "                bsync.ImpactOnOccupantComfort(\"Low\"),\n",
+    "                bsync.EstimatedCost(\"Medium\"),\n",
+    "                bsync.EstimatedAnnualSavings(\"Medium\"),\n",
+    "                bsync.EstimatedROI(\"High\"),\n",
+    "                bsync.SimpleImpactAnalysis.Priority(\"Medium\")\n",
+    "            ),\n",
+    "            bsync.AnnualSavingsAverageGHGEmissions(600.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:1290]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 12]
```diff
@@ -0,0 +1,1435 @@
...
+    "            bsync.AnnualSavingsMarginalGHGEmissions(210.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:1291]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 13]
```diff
@@ -0,0 +1,1435 @@
...
+    "            bsync.EndTimestamp(end_dt),\n",
+    "            bsync.IntervalFrequency(\"Month\"),\n",
+    "            bsync.IntervalReading(float(val)),\n",
+    "            bsync.ResourceUseID(IDref=resource_use_id),\n",
+    "            ID=my_id\n",
+    "        )\n",
+    "        monthly.append(ts)\n",
+    "        \n",
+    "        # for this simple example we are only handling electricity and ng\n",
+    "        if tsrq == 'Energy':\n",
```
Comment (JieXiong9119) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:921]:
```suggestion
    "        if tsrq == 'Greenhouse Gas Emissions':\n",
    "            ghg_ids.append(my_id)\n",
    "        if tsrq == 'Energy':\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 14]
```diff
@@ -0,0 +1,1435 @@
...
+    "monthly_ghg = [250.0, 240.0, 260.0, 250.0, 260.0, 230.0, 280.0, 270.0, 260.0, 250.0, 240.0, 250.0]\n",
+    "monthly_ng = [5.7, 4.01, 0.58, 0.4, 0.02, 0, 0, 0, 0, 0.01, 0.36, 6.08]\n",
+    "monthly_elec_peak = [15.42, 15.5, 16.25, 16.65, 18.56, 20.01, 20.82, 21.12, 20.42, 20.08, 17.4, 16.3]\n",
+    "\n",
+    "# costs data\n",
+    "monthly_elec_cost = [520.48, 451.53, 464.83, 389.43, 403.84, 416.49, 445.29, 461.68, 419.64, 406.87, 439.23, 484.14]\n",
+    "monthly_ng_cost = [30.44, 21.41, 3.10, 2.14, 0.11, 0.00, 0.00, 0.00, 0.00, 0.05, 1.92, 32.47]\n",
+    "\n",
+    "elec_ids = []\n",
+    "ng_ids = []\n",
```
Comment (JieXiong9119) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:888]:
```suggestion
    "ghg_ids = []\n",
    "ng_ids = []\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 15]
```diff
@@ -0,0 +1,1435 @@
...
+    "        # for this simple example we are only handling electricity and ng\n",
+    "        if tsrq == 'Energy':\n",
+    "            if 'Electricity' in resource_use_id:\n",
+    "                elec_ids.append(my_id)\n",
+    "            else:\n",
+    "                ng_ids.append(my_id)\n",
+    "    return monthly\n",
+    "    \n",
+    "elec_ts = create_monthly(monthly_elec, elec_ru['ID'], 2019)\n",
+    "ghg_ts = create_monthly(monthly_ghg, elec_ru['ID'], 2019, tsrq=\"Greenhouse Gas Emissions\")\n",
```
Comment (JieXiong9119) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:929]:
```suggestion
    "ghg_ts = create_monthly(monthly_ghg, ghg_ru['ID'], 2019, tsrq=\"Greenhouse Gas Emissions\")\n",
```

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 16]
```diff
@@ -0,0 +1,1435 @@
...
+    "            start_month = 12\n",
+    "            end_month = 1\n",
+    "            end_year += 1\n",
+    "        else:\n",
+    "            start_month = i % 12\n",
+    "            end_month = start_month + 1\n",
+    "        start_dt = datetime(start_year, start_month, 1)\n",
+    "        end_dt = datetime(end_year, end_month, 1)\n",
+    "\n",
+    "        my_id = f\"TS-{resource_use_id}-{tsrq}-{start_month}\"\n",
```
Comment (JieXiong9119) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:907]:
```suggestion
    "        if tsrq == 'Greenhouse Gas Emissions':\n",
    "            my_id = f\"TS-{resource_use_id}-{start_month}\"\n",
    "        else:\n",
    "            my_id = f\"TS-{resource_use_id}-{tsrq}-{start_month}\"\n",
```
The `resource_use_id` already adds `GreenhouseGasEmissions` to the `ID`, and `tsrq` would introduce spaces into `ID` when it's related to ghg.

File: docs/notebooks/bsync_examples/Small-Office-GHG.ipynb [hunk 17]
```diff
@@ -0,0 +1,1435 @@
...
+    "            bsync.StartTimestamp(start_dt),\n",
+    "            bsync.EndTimestamp(end_dt),\n",
+    "            bsync.IntervalFrequency(\"Month\"),\n",
+    "            bsync.IntervalReading(float(val)),\n",
+    "            bsync.ResourceUseID(IDref=resource_use_id),\n",
+    "            ID=my_id\n",
+    "        )\n",
+    "        monthly.append(ts)\n",
+    "        \n",
+    "        # for this simple example we are only handling electricity and ng\n",
```
Comment (JieXiong9119) [inline docs/notebooks/bsync_examples/Small-Office-GHG.ipynb:920]:
```suggestion
```

**Review by JieXiong9119 — DISMISSED** [review]
Some modifications suggestions based on validation results and my thoughts.

**Comment by nllong** [comment]
Any reason not to add this to the existing `Small-Office-Level-1` notebook instead of creating a new one? It is still appropriate to have this data in a Level-1 audit?

**Comment by laurenliz22** [comment]
> Any reason not to add this to the existing `Small-Office-Level-1` notebook instead of creating a new one? It is still appropriate to have this data in a Level-1 audit.

That is what we were not sure about, if it was still appropriate to have in a Level-1 audit.  Since it is, we can add it to the current notebook

**Comment by nllong** [comment]
> > Any reason not to add this to the existing `Small-Office-Level-1` notebook instead of creating a new one? It is still appropriate to have this data in a Level-1 audit.
> 
> That is what we were not sure about, if it was still appropriate to have in a Level-1 audit. Since it is, we can add it to the current notebook

Yeah, I would just add it to the existing one. Sorry, I know merging ipynb files is painful!

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 18]
```diff
@@ -1253,13 +1395,20 @@
     "        bsync.EnergyCost(5395.),\n",
     "        bsync.EnergyCostIndex(0.98), # $/ft2\n",
     "        ID=\"AllResourceTotal-1\"\n",
+    "    ),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1398]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 19]
```diff
@@ -1253,13 +1395,20 @@
...
+    "    bsync.AllResourceTotal(\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1399]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 20]
```diff
@@ -1253,13 +1395,20 @@
...
+    "        bsync.AllResourceTotal.AnnualAverageGHGEmissions(10000.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1400]:
```suggestion
    "        
```

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 21]
```diff
@@ -1253,13 +1395,20 @@
...
+    "        bsync.AllResourceTotal.AnnualMarginalGHGEmissions(8000.0),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1401]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 22]
```diff
@@ -1253,13 +1395,20 @@
...
+    "        bsync.AllResourceTotal.AnnualGHGEmissionIntensity(0.123),\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1402]:
```suggestion
```

File: docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb [hunk 23]
```diff
@@ -1253,13 +1395,20 @@
...
+    "        ID=\"AllResourceTotal-2\"\n",
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Small-Office-Level-1.ipynb:1403]:
```suggestion
```

**Pull request merged**